src/include/meta.h

```diff
@@ -47,6 +47,8 @@
 #define WT_DISAGG_METADATA_MAIN_PAGE_ID 1                        /* Page ID for the main metadata */
 #define WT_DISAGG_METADATA_MAX_PAGE_ID 1                         /* Max page ID in the metadata */
 
+#define WT_DISAGG_KEY_PROVIDER_TABLE_ID 2 /* Table ID for key encryption keys */
```

Comment:
Should it go next to the metadata table id definition if this is the reason why it's 2?